app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt

```diff
@@ -146,6 +146,10 @@ class MessageActivity : AppCompatActivity() {
                 }
 
                 R.id.textRadio -> {
+                    val inputManager: InputMethodManager = this.getSystemService(Context.INPUT_METHOD_SERVICE)
+                            as InputMethodManager
+                    inputManager.showSoftInput(content, 1)
```

Comment:
Never use integers as the second parameter. Rather use InputMethodManager.SHOW_IMPLICIT Which is a constant of InputMethodManager. But as you want to force the keyboard, use InputMethodManager.SHOW_FORCED

Reply:
also, instance of inputmanager is being create 3 times inside MessageActivity.kt , just define it  outside instead of intializing it in every scope

Reply:
I'll make these changes.